ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -65,6 +65,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: e2e-aws-ipi-destructive-p3
+  cron: 0 0 */28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
```

Comment:
@liangxia  please add`TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial&;~DisconnectedOnly&` with comment https://github.com/openshift/release/pull/30585#discussion_r925117082

Note: `~ChkUpgrade&;~NonPreRelease&;Serial&;~DisconnectedOnly&` is OK because Serial includes Disruptive for golang.

of course, it is also OK to use `~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~DisconnectedOnly&`